### PR TITLE
tweak README

### PR DESCRIPTION
**Pull request by ralphtheninja**
get rid of ABI section and unify that with Build, it was confusing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,13 +18,32 @@ $ npm install -g prebuild
...
+
+Optionally, to always build for the above versions you can add the following to `~/.prebuildrc`.
+
+``` ini
+prebuild[] = 0.10.42
+prebuild[] = 0.12.10
+prebuild[] = 4.3.0
 ```
 
+Note that `~/.prebuildrc` is a global configuration, i.e. when building *any* native module on your system. If this is not what you want you should consider adding a `.prebuildrc` to your project. This way the module determines which version it supports rather than a global setting.
```
Comment (juliangruber) [inline README.md]: should we make the local `.prebuildrc` case the default?

Comment (ralphtheninja) [inline README.md]: Yes! Good call :) Maybe not even mention the global setting?

Comment (juliangruber) [inline README.md]: sounds good to me!

Comment (ralphtheninja) [inline README.md]: We could maybe mention that you can use `~/.prebuildrc` for storing your token, which is a setting you really want globally

Comment (juliangruber) [inline README.md]: true, i think that's the common use case

**Comment by ralphtheninja** [comment]
I'll squash and merge on +1

**Comment by mafintosh** [comment]
Looks good to me

**Comment by ralphtheninja** [comment]
3.0.1

**Pull request merged**